### PR TITLE
MGMT-10705: Return multipath device types

### PR DESCRIPTION
**Pull request by avishayt**

### CODE REVIEW

File: src/inventory/disks.go [hunk 1]
```diff
@@ -189,6 +197,19 @@ func (d *disks) isMultipath(disk *ghw.Disk) bool {
 	return false
 }
 
+func (d *disks) getHolders(diskName string) string {
+	dir := fmt.Sprintf("/sys/block/%s/holders", diskName)
+	files, err := d.dependencies.ReadDir(dir)
+	if err != nil || len(files) == 0 {
+		return ""
```
Comment (tsorya) [inline src/inventory/disks.go:204]: Should not we log err? though not sure why it should happened but better to have some log in any case

Comment (avishayt) [reply src/inventory/disks.go:209]: Fixed

**Pull request merged**